server/schemas/users.js

```diff
@@ -0,0 +1,20 @@
+let mongoose = require('mongoose');
+//Users SCHEMA 
+let usersSchema = new mongoose.Schema({
+    id: Number,
+    account_type: String,
+    first_name: String,
+    last_name: String,
+    address: String,
+    city: String,
+    region: String,
+    zipcode: String,
+    phone: Number,
+    email: String,
+    hashed_password: String,
+    image_src: String,
+    status: String
+});
```

Comment:
This has too much data in this - keep it smaller and contain only what we need for the beginning of this application